.github/workflows/validate.yml

```diff
@@ -4,28 +4,25 @@ on:
   - pull_request
   - push
 
+permissions: {}
+
 jobs:
   validate:
     name: Lint & Test
+    permissions:
+      checks: write
     runs-on: ubuntu-latest
-    env:
-      CI: true
     steps:
       - name: Check out repo
-        uses: actions/checkout@master
-        with:
-          lfs: true
+        uses: actions/checkout@v3
 
       - name: Set up Node.js 16.x
-        uses: actions/setup-node@master
+        uses: actions/setup-node@v3
         with:
           node-version: 16.x
 
       - name: Install dependencies
         run: yarn install --frozen-lockfile
 
-      - name: Test
```

Comment:
Should we call the step just Lint instead of Lint & Test if we're only running lint?